xenit-alfresco/templates/acs/acs-deployment.yaml

```diff
@@ -169,6 +181,19 @@ spec:
             claimName: {{ .name }}-pvc
       {{- end }}
       {{- end }}
+      {{- if .Values.acs.configMapVolumes }}
+      {{- range .Values.acs.configMapVolumes }}
+        - name: {{ .name }}
+          configMap:
+            defaultMode: 420
+            name: ldap1-ad-auth-config
```

Comment:
this should be configurable , isn't better to just include the yaml instead of each field

Reply:
the name was indeed an oversight. I added the value in the readme but not in the yaml. Thanks for pointing that out.
Since the data needs to be spread in 2 places I dont know if yaml is the best option here though. Now we also guarantee that the name of the volume and the mount match